src/main/java/com/epam/spring/library/model/Author.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.spring.library.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+@Builder
+public class Author implements Entity {
+    private int id;
+    private String defaultName;
+    private Language defaultLanguage;
+    @Builder.Default
+    private Map<Language, String> i18Names = new HashMap<>();
```

Comment:
`nameTranslations` or `nameByLanguage` seams more reasonable name for a variable

Reply:
also can be declared final

Reply:
> also can be declared final

Yes, this map definitely can and so should be declared final, thanks. Other fields can not as they are used in update functions. I just clarify it as I'm not sure did you asked about them or not. The same is with the Book entity below.